ckanext/benap/benap_schema.json

```diff
@@ -262,16 +262,30 @@
       "required": "true"
     },
     {
-      "field_name": "publisher_name",
+      "field_name": "publisher_firstName",
       "label": {
-        "en": "Name [publisher]",
+        "en": "First name [publisher]",
+        "fr": "Prénom [éditeur]",
+        "nl": "Voornaam [uitgever]",
+        "de": "Vorname [Herausgeber]"
+      },
+      "form_placeholder": "Zenodotus",
+      "required": "true",
+      "validators": "benap_is_choice_null"
+    },
+    {
+      "field_name": "publisher_surname",
+      "label": {
+        "en": "Surname [publisher]",
         "fr": "Nom [éditeur]",
-        "nl": "Naam [uitgever]",
-        "de": "Name [Herausgeber]"
+        "nl": "Familienaam [uitgever]",
+        "de": "Familienname [Herausgeber]"
       },
-      "form_placeholder": "John Doe",
-      "required": "true"
+      "form_placeholder": "Of Ephesus",
+      "required": "true",
+      "validators": "benap_is_choice_null"
```

Comment:
validator benap_is_choice_null is niet nodig aangezien het een verplicht veld is.